content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -15,17 +15,411 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
+
+With the addition of the new `Schedules` resource and the `world.run_schedule(schedule_label: impl ScheduleLabel)`API it's more :sparkles: ergonomic :sparkles: than ever.
+
+```rust!
+
+// A schedule!
+let mut my_schedule = Schedule::new();
+schedule.add_system(my_system);
+
+// A schedule label for it
+#[derive(ScheduleLabel, Debug, Hash, PartialEq, Eq, Clone)]
+struct MySchedule;
+
+// An exclusive system to run this schedule
+fn run_my_schedule(world: &mut World) {
+    while very_complex_logic(){
+        world.run_schedule(MySchedule);
+    }
+}
+
+// Behold the ergonomics
+app
+    .add_schedule(MySchedule, my_schedule)
+    .add_system(run_my_schedule);
+```
+
+Bevy uses this pattern for five rather different things at 0.10 release:
+
+1. **Startup systems:** these now live in their own schedule, which is run once at the start of the app.
+2. **Fixed timestep systems:** another schedule?! The exclusive system that runs this schedule accumulates time, running a while loop that repeatedly runs `CoreSchedule::FixedTimestep` until all of the accumulated time has been spent.
+3. **Entering and exiting states:** a bonanza of schedules. Each collection of systems that runs logic to enter and exit a state variant is stored in its own schedule, which are called based on the change in state in the `apply_state_transitions::<S>` exclusive system.
+4. **Rendering:** all rendering logic is stored in its own schedule to allow it to run asynchronously relative to gameplay logic.
+5. **Controlling the outermost loop:** in order to handle the "startup schedule first, then main schedule" logic, we wrap it all up in a minimal overhead `CoreSchedule::Outer` and then run our schedules as the sole exclusive system there.
+
+Follow the bread crumbs starting at [`CoreSchedule`](https://dev-docs.bevyengine.org/bevy/app/enum.CoreSchedule.html) for more info.
+
+## Simpler Run Conditions
+
+Systems may have any number of run conditions (and inherit them from the sets they belong to), but will only run if all of their run conditions return `true`.
+Run criteria have been renamed to the clearer **run conditions**, which can be constructed out of any read-only system that returns `bool`.
+
+With a new blessed pattern for complex control flow, we can finally get rid of looping run criteria. [`ShouldRun::YesAndCheckAgain`](https://docs.rs/bevy/0.9.1/bevy/ecs/schedule/enum.ShouldRun.html) was not exactly straightforward to reason about, either for engine devs or users. It's always a bad sign when your bool-like enums have four possible values.
+
+If you crave that powerful, complex control flow: use the "schedules in exclusive systems" pattern listed above.
+For the other 99% of use cases, enjoy the simpler `bool`-based run conditions.
+
+```rust!
+// Let's make our own run condition
+fn contrived_run_condition(query: Query<&Life, With<Player>>, score: Res<Score>) -> bool{
```

Comment:
Nit: missing a space before the bracket at the end of the line

Reply:
```suggestion
fn contrived_run_condition(query: Query<&Life, With<Player>>, score: Res<Score>) -> bool {
```